packages/e2e/local-network/scripts/download-config.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script is for development purposes only. It will download IOG-maintained config files so we can compare them with current config files.
```

Comment:
Just need to capture this as follow-up work as per my feedback in https://github.com/input-output-hk/cardano-js-sdk/pull/305#pullrequestreview-1024250258

Reply:
I will start with this as my first task when go back to complete the work on [ADP-1899](https://input-output.atlassian.net/browse/ADP-1899)